gel/ai/core.py

```diff
@@ -45,25 +45,26 @@ class BaseRAGClient:
 
     def __init__(
         self,
-        client: typing.Union[gel.Client, gel.AsyncIOClient],
+        info: gel.ConnectionInfo,
         options: types.RAGOptions,
         **kwargs,
     ):
-        pool = client._impl
-        host, port = pool._working_addr
-        params = pool._working_params
```

Comment:
Not a blocker, but I think this diff could be a little smaller and focus if you kept this alias like:

```py
        params = info.params
```

I think that would reduce the downstream changes in the rest of this `__init__` def?